modules/standard/Sort.chpl

```diff
@@ -414,10 +414,22 @@ inline proc chpl_compare(a:?t, b:t, comparator:?rec) {
     return compareByPart(a, b, comparator);
   } else {
     // never reached, chpl_check_comparator will catch this
```

Comment:
It looks like this comment is not true, should it be removed?

Reply:
its sadly not true and should definitely should be removed :smile: